test/unit/lib/postcode_sanitizer_test.rb

```diff
@@ -0,0 +1,22 @@
+require 'test_helper'
+
+class PostcodeSanitizerTest < ActiveSupport::TestCase
+
+  context "postcodes come through to imminence" do
+
+    should "strip trailing spaces from entered postcodes" do
+      assert_equal "WC2B 6NH", PostcodeSanitizer.sanitize("WC2B 6NH ")
+    end
+
+    should "strip non-alphanumerics from entered postcodes" do
+      assert_equal "WC2B 6NH", PostcodeSanitizer.sanitize("WC2B   -6NH]")
+    end
+
+    should "transpose O/0 and L/1 if necessary" do
```

Comment:
I noticed this test says `L/1` but it's actually testing `I/1`. Does it also substitute for `L` and `l`?